wlog/levels.go

```diff
@@ -50,3 +50,34 @@ func (l *LogLevel) UnmarshalText(b []byte) error {
 		return fmt.Errorf("invalid log level: %q", string(b))
 	}
 }
+
+func (l LogLevel) Enabled(other LogLevel) bool {
```

Comment:
I guess it likely doesn't matter given that `LogLevel` is a string in memory, but would it make more sense for this to be a pointer receiver?

Reply:
I prefer the value receiver when the method is read-only because (1) it follows a principle of passing the minimum amount of data the function needs (it doesn't care about the address from the caller) and (2) we would just end up doing `switch *l` which doesn't add much

Reply:
Got it, thanks!

Separate nit: can we add newline between previous struct definition and this function definition (line 53 and 54)?